miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditSampleController.java

```diff
@@ -1011,6 +1030,7 @@ protected void writeConfiguration(ObjectMapper mapper, ObjectNode config) throws
         config.putPOJO("project", Dtos.asDto(project));
       }
       config.put(Config.DEFAULT_SCI_NAME, defaultSciName);
+      config.putPOJO(Config.BOX, box);
     }
   };
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this empty statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AEmptyStatementUsageCheck)